client/src/components/mixins/common-mixin.ts

```diff
@@ -59,4 +59,9 @@ export default class CommonMixin extends Vue {
     return [EntityType.XCR, EntityType.XUL, EntityType.RLC, EntityType.XLP, EntityType.XLL,
       EntityType.XCP, EntityType.XSO].includes(nr?.entity_type_cd)
   }
+
+  /** Scroll to given element Id */
+  private scrollTo (id) {
+    document.getElementById(id).scrollIntoView()
```

Comment:
I would add conditional chaining to avoid problems if the id isn't found.

Reply:
If possible, test this with Safari as well. I had to do the following to get scrolling working previously:
https://github.com/bcgov/business-filings-ui/blob/1c6a86ab2154646f806eb3d866bddd6beb56e32e/src/views/StandaloneDirectorsFiling.vue#L840

Reply:
ok, good call. Will do!

Reply:
So it navigates to the section, but safari doesn't support SMOOTH scrolling

Reply:
To support smooth scrolling for Safari it is either..
a) Some custom javascript
b) Add package to handle this.

Reply:
Ask UX / POs where to go from here.

Reply:
CSS transition?

Reply:
Copy that, will do!